### ISSUE TITLE
Very lightweight interactive viewer

### ISSUE THREAD

**Issue by mattwarkentin** [fepegar/torchio#726]
**🚀 Feature**

Hi @fepegar,

I love the new `Image.to_gif()` feature. It makes it really easy to create some cool visualizations. However, I would still love there to be some (very) lightweight interactive viewer. It is definitely a rabbit hole to implement anything too complex, and there are already good external viewer solutions, but I would love to be able to have a simple slider to flip through slices (along a specific axis). Maybe we can consider a new method `view()`.

```py
img = tio.ScalarImage(...)
img.view(axis=2) # one of 0, 1, or 2; possibly anatomical labels like 'axial', 'sagittal', 'coronal'
```

I don't currently know enough about the best way to implement such a slider plotting interface, but if there is interest I would be happy to look into this further and perhaps contribute a PR. What are your thoughts?

**Motivation**

It would make me happy?? :)

**Alternatives**

A very low-burden solution might be to use the `Show()` function from `SimpleITK`. This will open the image in an external viewer of choice (e.g. Fiji, ITKSnap, 3D Slicer, ImageJ, etc.). This would be fairly easy to implement with existing infrastructure. In fact, this might be useful to add independent of possibly implementing an in-house interactive viewer. Thoughts?

```py
class Image(...):
    ...
    def show(self, title, debugOn):
        sitk.Show(self.as_sitk(), title, debugOn)
```

**Comment by fepegar** [comment]
Hi, @mattwarkentin. It would make me happy too.

For option 1, I once implemented a little function. Feel free to try it and let me know if you're thinking of something like that: https://github.com/fepegar/miccai-educational-challenge-2019/blob/94067cdb99830ea21e51964ac9c20a34c28aaea4/visualization.py#L176-L202

Option 2 might be simpler but slower. I think we can implement option 2 quickly and then decide if we also want option 1. What do you think?

**Comment by fepegar** [comment]
`Image.show()` available in `v0.18.62`. Thanks, @mattwarkentin!

**Comment by fepegar** [comment]
Also, I learnt to write commit messages with co-authors, which is nice :)
